src/pages/filter/ui/query-builder/right-refiner-column/query-selected.tsx

```diff
@@ -44,6 +45,12 @@ export const QuerySelected = observer((): ReactElement => {
     filterStore.clearConditions()
   }
 
+  const isDisabledApplyButton =
+    isConditionsFetching ||
+    filteredVariantsCount == null ||
+    !conditions.length ||
+    isChanging
```

Comment:
Looks like isChanging should be opposite isn't it? The button should be disabled when isChanging is false?

Reply:
No, it is not. It is apply button for filter, so we can apply only when we don't have any unsaved changes.